dspace/config/launcher.xml

```diff
@@ -96,6 +96,19 @@
             <class>org.dspace.app.sitemap.GenerateSitemaps</class>
         </step>
     </command>
+    <command>
+        <name>get-counter-robots</name>
```

Comment:
Should this be named "update-counter-robots"?

I also wonder if this should install the file in the correct location?  Personally, I wouldn't know where to place this file.  Does it go under `[dspace]/spiders/` or `[dspace]/spiders/agents`?

If it doesn't put the file in the correct location, then we need to provide clear documentation on how to install this file.  Ideally, though, if we want people to be able to run this via a cron job, it'd be nice to have it install the file in the proper place.

Reply:
It's called "get-" rather than "update-" because it doesn't update; it gets, and lets you decide whether you want to update.

It never occurred to me that someone would trust a cron job to do the updating.  YOU decide to run it because you have learned that there's a new release from COUNTER.  YOU inspect the file and decide whether it's suitable.  YOU install it.  This seems to be what was asked for in #7020.

Maybe we need to have the discussion that Bram suggested, before finishing this up?  People seem to have very different ideas of how to consume the COUNTER robots list.

Reply:
@mwoodiupui : Yes, it sounds like we may need to discuss the goals of this PR further.  

My **assumption** was that we'd want this to be as easy as possible to help sites keep this list _updated at all times._   So, I was assuming it'd be as simply as a scheduled cron job that you could run to update it one a week (if changed) or similar.

Your approach is much more "manual", which I'm worried means it's not much of an improvement over the current process.  An Admin could just go to this site and download the latest file quickly to inspect it & install it.  This script currently seems to only help them in the download process (which could also be done via a `wget` command).

So, I think you are correct that our two goals for this script don't currently align.  I'd rather have an update script here, and I'm questioning whether a simple "get" script is useful when we could just tell people to run something like this:
```
wget https://github.com/atmire/COUNTER-Robots/blob/master/generated/COUNTER_Robots_list.txt
```